minutes/2018-10-12-node-interactive.md

```diff
@@ -0,0 +1,199 @@
+  * good start would be making pulls to community:
+    * update `mkdirp`, `rimraf`, based on `process.version`.
+    * _Michel_ also advocates this approach.
+
+* *Daniel*:
+  * how do we get adoption if only cutting edge Node.js should be using?
+  * perhaps backporting should be goal when possible?
+
+* *Bryan/Chris*:
+  * backporting is a pain in the butt, perhaps we just agree to polyfill;
+    see: readable stream.
+
+* *Ruy*:
+  * there are good aspects to not backporting features, drive more people
+    to adopt new Node.js versions.
+
+* *Ian*:
+  * because we're targetting CLI tools, perhaps we can be more aggressive
+    about pushing people to upgrade to new Node.js:
+    
+* *Daniel*:
+  * on topic of polyfills, `graceful-fs` is an example of 
+    _how not to polyfill_.
+
+* *Guy*:
+  * fs-extra is a better approach, doesn't mutate global fs object
+
+we seem to have reached relative consensus that:
+
+* we'll shim old library versions.
+* won't backport to old Node.js versions.
+* will use `process.version` for feature detecton.
+
+## What other sorts of features should we be working on?
+
+* *Chris*:
+  * argument parsing could be made better than `process.argv.slice(2)`.
+
+* *Bryan*:
+  * would like to see `execvp` implemented, this could be a good alternative
+    for how `npx` currently runs.
+
+* *Guy*:
+  * look at the libraries that perform Windows shimming (`cross-env`, `cross-spawn`).
+
+* *Ben*:
+  * I put together a list of these Windows issues which can be found here:
+  * https://github.com/bcoe/awesome-cross-platform-nodejs
+
+* *Chris*:
+  * what if we were smarter abut path generation, and `path.separator` was
+    no longer a thing you needed to do.
+
+* *Guy*:
+  * interesting to note that Deno does forward slashes by default.
+  
+* *Daniel*:
+  * whatever we build, let's avoid having runtime flags for it, e.g., forward
+    slashes being enabled by a flag.
+
+* *Ben*:
+  * _let's step up 30,000', and talk about some of the higher level goals of the
+    _Tooling Initiative :tm:_.
+
+## Action Items
+
+* *Chris*:
+  * let's finish building `rimraf`. (@bcoe, @boneskull).
+  * let's get `mkdirp` shimmed (@iansu).
+  * let's figure out how we're going to approach getting further user feedback (@bcoe).
+  * figure out how we're announcing future meetings; blog post? (@boneskull).
+
+* *Ben*:
+  * let's develop a roadmap we can share with TSC and other contributors. (@bcoe).
+  * let's schedule our first meeting:
+    * figure out roadmap first, use this to drive user feedback work.
+  * let's keep Windows folks in the loop, and make reducing Windows friction
+    an important initiative of this group (@michellopez)
+  * let's chat with TSC and figure out how our group interacts (@bcoe, @boneskull).
+
+* *Guy*:
+  * let's reach out to library maintainers, and see how they feel about getting
+    their work brought into core (@guybedord).
+    * this could be a great way to get more Node contributors; perhaps
```

Comment:
@dshaw it came up in the tooling meeting that it might be good to reach out to library maintainers who are interested in moving some of their work to Node.js core; might dovetail with community and mentorship work?